### PR TITLE
Update README.md

### LINKED ISSUE #5: Update Readme to tell more about the project.
Self explanatory

### PR DESCRIPTION
**Pull request by Tbaut**
closes #5

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,32 +1,58 @@
-# bridge-ui
+# Bridge-ui
 
-Empty project.
+This repo hosts a simple interface to send tokens using the [Rialto bridge](https://github.com/paritytech/parity-bridges-common) between an Ethereum network and a Substrate network.
+Metamask is required to sign Ethereum transactions.
 
-## Building and running on localhost
+The interface is available at: https://paritytech.github.io/bridge-ui/
```
Comment (tomusdrw) [inline README.md]: Maybe make it a link as well?

File: README.md [hunk 2]
```diff
@@ -1,32 +1,58 @@
-# bridge-ui
+# Bridge-ui
 
-Empty project.
+This repo hosts a simple interface to send tokens using the [Rialto bridge](https://github.com/paritytech/parity-bridges-common) between an Ethereum network and a Substrate network.
```
Comment (tomusdrw) [inline README.md]:
```suggestion
This repo hosts a simple interface to send tokens using the [Rialto bridge](https://github.com/paritytech/parity-bridges-common) between an Ethereum network and a Substrate network (one way).
```

File: README.md [hunk 3]
```diff
@@ -1,32 +1,58 @@
...
+Metamask is required to sign Ethereum transactions.
```
Comment (tomusdrw) [inline README.md]:
```suggestion
Metamask (connected to Rialto network) is required to sign Ethereum transactions. The interface automatically connects to Rialto's Substrate nodes (no configuration or plugin required).
```

File: README.md (hunk 1 above)
Comment (Tbaut) [reply README.md]: On Github it's changed into a link automatically actually. Making it explicit though.
```suggestion
The interface is available at: [https://paritytech.github.io/bridge-ui](https://paritytech.github.io/bridge-ui)
```

**Review by tomusdrw — APPROVED** [review]

**Pull request merged**